react/components/MicroData.js

```diff
@@ -0,0 +1,74 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { pathOr, path, map, sort, compose, head } from 'ramda'
+
+const lowestPriceInStockSeller = item => {
+  if (item.sellers.length) {
+    return sort((a, b) => a.commertialOffer && a.commertialOffer.availableQuantity > 0
```

Comment:
You could use better variable names, like `item` and `otherItem`.

Reply:
Guess I can rename those.
Well observed, thanks for the advice :)